snap/hooks/post-refresh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+if [ ! -d $SNAP_COMMON/host-hunspell ]; then
+  echo "No host-hunspell, skipping"
+  exit 0
+fi;
+DICPATH=$SNAP_COMMON/snap-hunspell
+
+if [ -d $DICPATH ]; then
+  # Cleanup on each refresh, ensure we have uptodate content
+  find $DICPATH -type l -name "*.dic" -or -name "*.aff" | xargs rm
+else
+  mkdir -p $DICPATH
+fi
+
+for dic in $(find $SNAP_COMMON/host-hunspell/ -name "*.dic");
+do
+  dic_file=$(basename $dic)
+  aff_file="${dic_file%%.dic}.aff"
+  ln -s $SNAP/usr/share/hunspell/${dic_file} $DICPATH/${dic_file}
+  ln -s $SNAP/usr/share/hunspell/${aff_file} $DICPATH/${aff_file}
+done;
+
+snapctl umount $SNAP_COMMON/host-hunspell || true
```

Comment:
See my other comment wrt when the `connect-plug-host-hunspell` hook is run. Shouldn't the `umount` operation be done in a corresponding `disconnect-plug-host-hunspell` hook?

Reply:
I'd need to ask people knowledgeable about that

Reply:
@oSoMoN My reasoning here was to unmount as soon as possible, doing it in the disconnect would be "too late" IMHO.

Reply:
Yes, I understand the rationale. Attempting to unmount in `disconnect-plug-host-hunspell` would be too late indeed.
But that means that after the first refresh `$SNAP_COMMON/host-hunspell` will be unmounted and never mounted back again, therefore updating the list of dictionaries will never happen, no?

Reply:
What is the drawback of doing unmount in disconnect hook ? Do you know ?